test/extended/single_node/topology.go

```diff
@@ -11,7 +13,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	e2e "k8s.io/kubernetes/test/e2e/framework"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
-	"strings"
```

Comment:
?

Reply:
Was fixed by my IDE, since its ordering of the imported pkgs is wrong here